test/teos/e2e/test_basic_e2e.py

```diff
@@ -187,6 +209,15 @@ def test_multiple_appointments_life_cycle(run_bitcoind):
         add_appointment(appointment)
         appointments_in_watcher += 1
 
+    available_slots -= 5
```

Comment:
This part can get a bit hard to maintain if some of the params of the function are changed. 

A better approach would be to decrease the number of available slots one by one in the previous for loop.